addons/pos_loyalty/static/src/overrides/models/loyalty.js

```diff
@@ -1127,15 +1127,19 @@ patch(Order.prototype, {
      */
     _getCheapestLine() {
         let cheapestLine;
-        for (const line of this.get_orderlines().filter((line) => !line.comboLines)) {
+        for (const line of this.get_orderlines().filter((line) => !line.comboParent)) {
+            let price = line.price;
+            if (line.isPartOfCombo()) {
+                price = line.getComboTotalPrice();
+            }
```

Comment:
```suggestion
            const price = line.isPartOfCombo() ? line.getComboTotalPrice() : line.price;
```
avoid mutation whenever possible